lightning/src/ln/msgs.rs

```diff
@@ -1608,6 +1637,7 @@ mod tests {
 			flags: if direction { 1 } else { 0 } | if disable { 1 << 1 } else { 0 } | if htlc_maximum_msat { 1 << 8 } else { 0 },
 			cltv_expiry_delta: 144,
 			htlc_minimum_msat: 1000000,
+			htlc_maximum_msat: if htlc_maximum_msat { OptionalField::Present(1) } else { OptionalField::Absent },
```

Comment:
Generally best to use a value which has more than one byte non-0 to test endianness.